### PR TITLE
Testing kubelet-density perfscale test

### PR DESCRIPTION
**Pull request by rsevilla87**
WIP, need to review with the team.
cc: @jtaleric

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.7.yaml [hunk 1]
```diff
@@ -138,6 +138,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-ovn
+- as: e2e-kubelet-density-step-registry
```
Comment (yuqi-zhang) [inline ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.7.yaml:141]: did you mean to add this to the MCO repo tests?

Comment (rsevilla87) [reply ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.7.yaml:141]: Yeah, have to double check with some team mates. For now I'm just only doing some tests

Comment (yuqi-zhang) [reply ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.7.yaml:141]: The MCO repo changes don't generally affect kubelet performance. I feel like this should be added somewhere else, once its ready

**Pull request closed without merging**